src/com/hms_networks/americas/sc/historicaldata/HistoricalDataQueueManager.java

```diff
@@ -270,6 +273,9 @@ private static long writeNewTime(String writeFile) throws IOException {
 
   /**
    * Get the historical log data for the specified tag groups within the next FIFO queue time span.
+   * The operations performed in this method use a significant amount of memory and it is
```

Comment:
would consume be a better word than use?

Reply:
changed